src/main/java/com/techeer/fashioncloud/global/dto/CustomPageRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.techeer.fashioncloud.global.dto;
+
+import com.techeer.fashioncloud.global.util.validation.ValidEnum;
+import io.swagger.v3.oas.annotations.Parameter;
+import jakarta.validation.constraints.Min;
+import lombok.*;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CustomPageRequest {
+
+    @Min(value = 0)
+    @Parameter(description = "(default: 0)")
+    private int page = 0;
+
+    @Min(value = 1)
+    @Parameter(description = "(default: 10)")
+    private int size = 10;
+
+    @ValidEnum(enumClass = Sort.Direction.class)
+    @Parameter(description = "(default: DESC)")
+    private Sort.Direction order = Sort.Direction.DESC;
+
+    @Parameter(description = "정렬 조건 (default: createdAt)")
+    private String[] sort = {"createdAt"};
+
+    public PageRequest of() {
+        return PageRequest.of(page, size, order, sort);
```

Comment:
```suggestion
    private int page = 0;

    @Min(value = 1)
    @Parameter(description = "(default: 10)")
    private final int size = 10;

    @ValidEnum(enumClass = Sort.Direction.class)
    @Parameter(description = "(default: DESC)")
    private final Sort.Direction order = Sort.Direction.DESC;

    @Parameter(description = "정렬 조건 (default: createdAt)")
    private final String[] sort = {"createdAt"};

    public PageRequest of() {
        return PageRequest.of(page, size, order, sort);
```

client에서 보낸 request는 변경이 될 가능성이 없고 변조가 되면 안됨으로 VO로 만드는 것이 좋을 것 같아요..!

Reply:
모든 request에 해당합니다!